### PR TITLE
Remove Alpha from CLI docs

### PR DESCRIPTION
**Pull request by DonNateR**

### CODE REVIEW

File: reference/cli/wpe/main/apps/list/README.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+# wpe apps list
 
 ## Description
 List all existing WP Engine Headless Applications
 
 ## Usage
 
 ```bash
-$ wpe alpha apps list [OPTIONS]
+$ wpe  apps list [OPTIONS]
```
Comment (wjohnsto) [inline reference/cli/wpe/main/apps/list/README.md:9]: Remove extra space between `wpe` and `apps`

File: reference/cli/wpe/main/apps/rebuild/README.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 
 ```bash
-$ wpe alpha apps rebuild $APP_NAME [OPTIONS]
+$ wpe apps rebuild $APP_NAME [OPTIONS]
 ```
 
 ## Extended Description
 
-The `wpe alpha apps rebuild` command will rebuild a specified WP Engine Headless Application. Use the [`wpe alpha apps list`](/reference/cli/wpe/alpha/apps/list) command to get a list of the apps you can rebuild.
+The `wpe  apps rebuild` command will rebuild a specified WP Engine Headless Application. Use the [`wpe  apps list`](/reference/cli/wpe//apps/list) command to get a list of the apps you can rebuild.
```
Comment (wjohnsto) [inline reference/cli/wpe/main/apps/rebuild/README.md:14]: Remove extra space between wpe and apps

File: reference/cli/wpe/main/apps/update/README.md [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
 
 ```bash
-$ wpe alpha apps update [OPTIONS]
+$ wpe apps update [OPTIONS]
 ```
 
 ## Extended Description
 
-The `wpe alpha apps update` command allows you to update an existing WP Engine Headless Application. Use the [`wpe alpha apps list`](/reference/cli/wpe/alpha/apps/list) command to get a list of the apps that you can update. You provide the GitHub repository you want to use, the environments you want to deploy, and the name of your app. Running the command will deploy the code from your repository to the corresponding environments.
+The `wpe  apps update` command allows you to update an existing WP Engine Headless Application. Use the [`wpe  apps list`](/reference/cli/wpe//apps/list) command to get a list of the apps that you can update. You provide the GitHub repository you want to use, the environments you want to deploy, and the name of your app. Running the command will deploy the code from your repository to the corresponding environments.
```
Comment (wjohnsto) [inline reference/cli/wpe/main/apps/update/README.md:14]: Remove extra space between wpe and apps

File: reference/cli/wpe/main/apps/delete/README.md [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
 
 ```bash
-$ wpe alpha apps delete $APP_NAME [OPTIONS]
+$ wpe apps delete $APP_NAME [OPTIONS]
 ```
 
 ## Extended Description
 
-The `wpe alpha apps delete` command allows you to remove an existing WP Engine Headless Application. Use the [`wpe alpha apps list`](/reference/cli/wpe/alpha/apps/list) command to get a list of the apps that you can delete. By default, this command will prompt you to re-enter the app name to ensure you are deleting the right app. You can override this with `--force`, which is useful if you are using automation to delete the app.
+The `wpe  apps delete` command allows you to remove an existing WP Engine Headless Application. Use the [`wpe  apps list`](/reference/cli/wpe//apps/list) command to get a list of the apps that you can delete. By default, this command will prompt you to re-enter the app name to ensure you are deleting the right app. You can override this with `--force`, which is useful if you are using automation to delete the app.
```
Comment (wjohnsto) [inline reference/cli/wpe/main/apps/delete/README.md:14]: Remove extra space between wpe and apps

File: reference/cli/wpe/main/context/get/README.md [hunk 5]
```diff
@@ -1,17 +1,17 @@
...
 
 ```bash
-$ wpe alpha context get [OPTIONS]
+$ wpe context get [OPTIONS]
 ```
 
 ## Extended Description
 
-The `wpe alpha context get` command will print the current account context that the WP Engine Headless CLI is using to deploy applications.
+The `wpe  context get` command will print the current account context that the WP Engine Headless CLI is using to deploy applications.
```
Comment (wjohnsto) [inline reference/cli/wpe/main/context/get/README.md:14]: Remove extra space between wpe and context

File: reference/cli/wpe/main/context/set/README.md [hunk 6]
```diff
@@ -1,17 +1,17 @@
...
 
 ```bash
-$ wpe alpha context set account $ACCOUNT_NAME
+$ wpe context set account $ACCOUNT_NAME
 ```
 
 ## Extended Description
 
-The `wpe alpha context set account` command allows you to set the account context that the WP Engine Headless CLI will use to deploy applications.
+The `wpe  context set account` command allows you to set the account context that the WP Engine Headless CLI will use to deploy applications.
```
Comment (wjohnsto) [inline reference/cli/wpe/main/context/set/README.md:14]: Remove extra space between wpe and context

File: reference/cli/wpe/main/envs/delete/README.md [hunk 7]
```diff
@@ -1,17 +1,17 @@
...
 
 ```bash
-$ wpe alpha envs delete $ENV_ID [OPTIONS]
+$ wpe envs delete $ENV_ID [OPTIONS]
 ```
 
 ## Extended Description
 
-The `wpe alpha envs delete` command will remove an existing deployed environment from an application.
+The `wpe  envs delete` command will remove an existing deployed environment from an application.
```
Comment (wjohnsto) [inline reference/cli/wpe/main/envs/delete/README.md:14]: Remove extra space between wpe and envs

File: reference/cli/wpe/main/envs/purge/README.md [hunk 8]
```diff
@@ -1,17 +1,17 @@
...
 
 ```bash
-$ wpe alpha envs purge $ENV_ID [OPTIONS]
+$ wpe envs purge $ENV_ID [OPTIONS]
 ```
 
 ## Extended Description
 
-The `wpe alpha envs purge` command allows you to purge the domain cache for a specified application environment.
+The `wpe  envs purge` command allows you to purge the domain cache for a specified application environment.
```
Comment (wjohnsto) [inline reference/cli/wpe/main/envs/purge/README.md:14]: Remove extra space between wpe and envs

File: reference/cli/wpe/main/envs/rebuild/README.md [hunk 9]
```diff
@@ -1,17 +1,17 @@
...
 
 ```bash
-$ wpe alpha envs rebuild $ENV_ID [OPTIONS]
+$ wpe envs rebuild $ENV_ID [OPTIONS]
 ```
 
 ## Extended Description
 
-The `wpe alpha envs rebuild` command allows you manually rebuild a specified environment for the application. It kicks off the same process that occurs when a new commit is pushed to git.
+The `wpe  envs rebuild` command allows you manually rebuild a specified environment for the application. It kicks off the same process that occurs when a new commit is pushed to git.
```
Comment (wjohnsto) [inline reference/cli/wpe/main/envs/rebuild/README.md:14]: Remove extra space between wpe and envs

File: reference/cli/wpe/main/logs/get/README.md [hunk 10]
```diff
@@ -1,17 +1,17 @@
...
 
 ```bash
-$ wpe alpha logs get [OPTIONS]
+$ wpe  logs get [OPTIONS]
 ```
 
 ## Extended Description
 
-The `wpe alpha logs get` command will print the current account logs that the WP Engine Headless CLI is using to deploy applications.
+The `wpe  logs get` command will print the current account logs that the WP Engine Headless CLI is using to deploy applications.
```
Comment (wjohnsto) [inline reference/cli/wpe/main/logs/get/README.md:14]: Remove extra space between wpe and logs

File: reference/cli/wpe/main/logs/get/README.md [hunk 11]
```diff
@@ -24,7 +24,7 @@ The `wpe alpha logs get` command will print the current account logs that the WP
 Let's assume the environment ID is `bmna3gedlzerwcb4vc04gssl` and the app name is `myapp`. Running the following command will print the logs for the environment:
 
 ```bash
-$ wpe alpha logs get -e bmna3gedlzerwcb4vc04gssl -a myapp
+$ wpe  logs get -e bmna3gedlzerwcb4vc04gssl -a myapp
```
Comment (wjohnsto) [inline reference/cli/wpe/main/logs/get/README.md:27]: Remove extra space between wpe and logs

File: reference/cli/wpe/main/logs/get/README.md [hunk 12]
```diff
@@ -1,17 +1,17 @@
...
+# wpe  logs get
 
 ## Description
 Get the application logs for the WP Engine Headless CLI.
 
 ## Usage
 
 ```bash
-$ wpe alpha logs get [OPTIONS]
+$ wpe  logs get [OPTIONS]
```
Comment (wjohnsto) [inline reference/cli/wpe/main/logs/get/README.md:9]: Remove extra space between wpe and logs

File: reference/cli/wpe/README.md [hunk 13]
```diff
@@ -3,14 +3,16 @@
 This section contains reference information on using WP Engine's command line
 client. Each command has a reference page along with samples.
 
+**NOTE:** With version 0.1.4 released March 18, 2021, Atlas CLI no longer requires the `alpha` syntax (such as `wpe alpha apps`of previous versions. If you are still using a version of Atlas CLI, run `npm i @wpengine/headless-cli -g` to use commands without `alpha`.
```
Comment (ikedracka) [inline reference/cli/wpe/README.md:6]: Isn't it missing "previous" or "older" in "If you are still using a version of Atlas CLI" though?

**Review by wjohnsto — APPROVED** [review]

**Pull request merged**